web/src/components/schedule/subcomponents/schedule_timeline/schedule_timeline.styles.js

```diff
@@ -0,0 +1,22 @@
+const scheduleTimelineStyles = {
+  sectionContainer: {
+    display: "flex",
+    justifyContent: "center",
+  },
+  section: {
+    backgroundColor: "white",
```

Comment:
Use theme here?

Reply:
The white isn't true white, it's actually a light gray that matches the rest of the background. I wanted this timeline card to stand out some